rgd/geodata/tests/test_fmv.py

```diff
@@ -1,21 +1,37 @@
+import shutil
+
 import pytest
 
 from rgd.geodata.datastore import datastore
 from rgd.geodata.models.fmv.base import FMVEntry
+from rgd.geodata.models.mixins import Status
 
 from . import factories
 
+NO_KWIVER = not shutil.which('kwiver') or not shutil.which('ffmpeg')
```

Comment:
If we want to keep these tests more isolated, we could mock the subprocess call. It's up to you though; maybe it's easy enough to install these things in CI.

Reply:
It is easy enough to install these in CI but maybe not on the developer's machine. I figured if someone doesn't have this installed, its easier to skip locally and run in CI

Reply:
How about we change this to allow an environment variable to skip these tests, for those who wish to run natively on other platforms? The advantage of that is that we won't ever accidentally skip them in CI if they're explicitly opt-out.